shortfin/setup.py

```diff
@@ -361,8 +367,25 @@ def populate_built_package(abs_dir):
 setup(
     name="shortfin",
     version=f"{PACKAGE_VERSION}",
-    description="Shortfin native library implementation",
     author="SHARK Authors",
+    description="shortfin - SHARK inference library and serving engine",
+    long_description=README,
+    long_description_content_type="text/markdown",
+    license="Apache-2.0",
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
+        "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.13",
+    ],
+    project_urls={
+        "repository": "https://github.com/nod-ai/SHARK-Platform",
+        "documentation": "https://shortfin.readthedocs.io/en/latest/",
+    },
+    python_requires=">=3.10",
```

Comment:
We may want to add this to the `pyproject.toml` file to have these information in the same place as for the `sharktank` and `shark-ai` package.